### ISSUE TITLE
BitPivot component missing features

### ISSUE THREAD

**Issue by maryamhdr** [bitfoundation/bitframework#795]
Some HTML attributes like `aria-labelledby`, `aria-hidden`, and some attributes relevant to the pivot item button should be added to the BitPivot component.
